site2/docs/getting-started-clients.md

```diff
@@ -6,53 +6,26 @@ sidebar_label: Use Pulsar with client libraries
-
-Library | Purpose
-:-------|:-------
-[`org.apache.pulsar.client.api`](/api/client) | The [Pulsar Java client](client-libraries-java.md) is used to produce and consume messages on Pulsar topics.
-[`org.apache.pulsar.client.admin`](/api/admin) | The Java client for the [Pulsar admin interface](admin-api-overview.md).
-
-
-## Go client
-
-For a tutorial on using the Pulsar Go client, see [Pulsar Go client](client-libraries-go.md).
-
-
-## Python client
-
-For a tutorial on using the Pulsar Python client, see [Pulsar Python client](client-libraries-python.md).
-
-There are also [pdoc](https://github.com/BurntSushi/pdoc)-generated API docs for the Python client [here](/api/python).
-
-## C++ client
-
-For a tutorial on using the Pulsar C++ clent, see [Pulsar C++ client](client-libraries-cpp.md).
-
-There are also [Doxygen](http://www.doxygen.nl/)-generated API docs for the C++ client [here](/api/cpp).
-
-## Feature Matrix
+- [Java client](client-libraries-java.md)
+- [Go client](client-libraries-go.md)
+- [Python client](client-libraries-python.md)
+- [C++ client](client-libraries-cpp.md)
+- [Node.js client](client-libraries-node.md)
+- [WebSocket client](client-libraries-websocket.md)
+
+## Feature matrix
 Pulsar client feature matrix for different languages is listed on [Client Features Matrix](https://github.com/apache/pulsar/wiki/Client-Features-Matrix) page.
 
-## Thirdparty Clients
+## Third-party clients
 
-Besides the official released clients, there are also multiple projects on developing a Pulsar client in different languages.
+Besides the official released clients, multiple projects on developing Pulsar clients are available in different languages.
 
 > If you have developed a new Pulsar client, feel free to submit a pull request and add your client to the list below.
 
 | Language | Project | Maintainer | License | Description |
 |----------|---------|------------|---------|-------------|
-| Go | [pulsar-client-go](https://github.com/Comcast/pulsar-client-go) | [Comcast](https://github.com/Comcast) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | A native golang client |
-| Go | [go-pulsar](https://github.com/t2y/go-pulsar) | [t2y](https://github.com/t2y) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | |
+| CGo | [pulsar-client-go](https://github.com/apache/pulsar/tree/master/pulsar-client-go) | [Apache Pulsar](https://github.com/apache/pulsar) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | CGo client that depends on C++ client library |
+| Go | [pulsar-client-go](https://github.com/apache/pulsar-client-go) | [Apache Pulsar](https://github.com/apache/pulsar) | [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0) | A native golang client |
```

Comment:
These libraries are not 3rd party but maintained by the project.

Reply:
@merlimat thank you very much for your comment. Updated, PTAL again.